docs/build/configure/triggers.md

```diff
@@ -209,32 +209,79 @@ The following attributes are available for triggers:
 5. Write your cloud function or lambda to process the request from `viam-server`.
    The following example function prints the received headers:
 
-   ```python {class="line-numbers linkable-line-numbers"}
-   import functions_framework
-   import requests
-   import time
+   {{< tabs >}}
+   {{% tab name="Flask" %}}
+
+```python {class="line-numbers linkable-line-numbers" }
+from flask import Flask, request
+
+app = Flask(__name__)
+
+
+@app.route("/", methods=['GET', 'POST'])
+def hello_http_get():
```

Comment:
add post too?

Reply:
Oh it'll do both but I can rename the function to not mention get 😅